chainer/datasets/fashion_mnist.py

```diff
@@ -8,6 +8,28 @@
 from chainer.datasets._mnist_helper import preprocess_mnist
 
 
+fashion_mnist_labels = ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat',
+                        'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+
+def get_fashion_mnist_labels(labels):
+    """Given a list of fashion-MNIST labels, returns name list of those labels.
+
+    Args:
+        labels (list): List of labels of image values, numbers from 0-9.
+
+    Returns:
+        List of string values of English names of the image labels.
+
+    """
+
+    assert isinstance(labels, list)
+    for l in labels:
+        assert range(10).index(l) >= 0
+
+    return [fashion_mnist_labels[l] for l in labels]
```

Comment:
I thought just `return _fashion_mnist_labels` is enough (users can then use it as they like if they need a subset.)

Reply:
How about this comment?

Reply:
@Crissman

Reply:
Thanks. Didn't see these comments since they were hidden.

If returning the list, wouldn't it be better to just make it available as part of the import, like:
```
fashion_mnist_labels = ['T-shirt/top', ...]
```

Why bother making it a function at all?

Reply:
Because it's aligned with other methods. (`get_ptb_words` v.s. `get_ptb_words_vocabulary` and `get_fashion_mnist` v.s. `get_fashion_mnist_labels`).

Reply:
👍